File: app/src/main/java/ua/com/foxminded/locationtrackera/model/locations/dao/LocationsDao.java
package ua.com.foxminded.locationtrackera.model.locations.dao;

import java.util.List;

import ua.com.foxminded.locationtrackera.model.locations.UserLocation;

public interface LocationsDao {

    void saveLocation(UserLocation userLocation);

    List<UserLocation> getAllLocations();

    void deleteAllLocation();
}

File: models/src/main/java/ua/com/foxminded/locationtrackera/models/locations/dao/LocationsDao.kt
package ua.com.foxminded.locationtrackera.models.locations.dao

import ua.com.foxminded.locationtrackera.models.locations.UserLocation

interface LocationsDao {

    fun saveLocation(userLocation: UserLocation)

    fun getAllLocations(): List<UserLocation>

    fun deleteAllLocation()
}